llvm/test/SYCLLowerIR/esimd_global_undef.ll

```diff
@@ -0,0 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -ESIMDLowerVecArg -S | FileCheck %s
+
```

Comment:
please add a comment what this test checks

Reply:
done